server/jetstream_test.go

```diff
@@ -9279,6 +9279,51 @@ func TestJetStreamPullConsumerMaxWaiting(t *testing.T) {
 	}
 }
 
+func TestJetStreamChangeConsumerType(t *testing.T) {
+	s := RunBasicJetStreamServer(t)
+	defer s.Shutdown()
+
+	nc, js := jsClientConnect(t, s)
+	defer nc.Close()
+
+	_, err := js.AddStream(&nats.StreamConfig{Name: "TEST", Subjects: []string{"test.*"}})
+	require_NoError(t, err)
+
+	// create pull consumer
+	_, err = js.AddConsumer("TEST", &nats.ConsumerConfig{
+		Name:      "pull",
+		AckPolicy: nats.AckExplicitPolicy,
+	})
+	require_NoError(t, err)
+
+	// cannot update pull -> push
+	_, err = js.UpdateConsumer("TEST", &nats.ConsumerConfig{
+		Name:           "pull",
+		AckPolicy:      nats.AckExplicitPolicy,
+		DeliverSubject: "foo",
+	})
+	if !strings.Contains(err.Error(), "can not update pull consumer to push based") {
```

Comment:
Minor nit but assertions like `require_Equals(err.Error(), "can not update pull consumer to push based")` are better not just because they are cleaner but because they properly integrate with the Antithesis SDK assertions.

Reply:
done, thanks 👍